src/main/app/controller/courts/ContactsController.ts

```diff
@@ -25,7 +25,7 @@ export class ContactsController {
 
     if (!contacts) {
       await req.scope.cradle.api.getContacts(slug)
-        .then((value: Contact[]) => contacts = value)
+        .then((value: Contact[]) => contacts = value.map(c => { c.isNew = false; return c; }))
```

Comment:
Can we add a comment in to mention what this achieves? 

Looks like it's returning a populated contact of some sort lol, but not too sure why haha